docs/general/contributing/development.md

```diff
@@ -187,13 +187,16 @@ Run each command on a separate line. The container we'll test in is named `jftes
 
 ```sh
 docker exec -ti jftest bash
-apt-get update && apt-get install git gnupg wget apt-transport-https curl autoconf g++ make libpng-dev gifsicle automake libtool make gcc musl-dev nasm
-wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg && mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/
-wget -q https://packages.microsoft.com/config/debian/10/prod.list && mv prod.list /etc/apt/sources.list.d/microsoft-prod.list
-apt-get update && apt-get install dotnet-sdk-7.0 npm
+apt-get update && apt-get install -y git gnupg wget apt-transport-https curl autoconf g++ make libpng-dev gifsicle automake libtool make gcc musl-dev nasm ca-certificates
+curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /usr/share/keyrings/microsoft-prod.gpg
```

Comment:
Getting the following error when stored in the suggested location on the latest `bookworm-slim`:

```
# curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /etc/apt/keyrings/microsoft-prod.gpg
# apt-get update && apt-get install -y dotnet-sdk-8.0

Get:1 https://packages.microsoft.com/debian/12/prod bookworm InRelease [3617 B]
Err:1 https://packages.microsoft.com/debian/12/prod bookworm InRelease      
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EB3E94ADBE1229CF
Hit:2 http://deb.debian.org/debian bookworm InRelease
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Hit:4 http://deb.debian.org/debian-security bookworm-security InRelease
Reading package lists... Done
W: GPG error: https://packages.microsoft.com/debian/12/prod bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY EB3E94ADBE1229CF
E: The repository 'https://packages.microsoft.com/debian/12/prod bookworm InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

Keeping it in `/usr/share` works and no errors.

Reply:
I checked on internet and my bad, thanks for verifying !